src/cmd/run.js

```diff
@@ -276,11 +279,12 @@ export class ExtensionRunner {
   firefoxApp: typeof defaultFirefoxApp;
   firefox: string;
   customPrefs: FirefoxPreferences;
+  startUrl: ?string = null;
```

Comment:
No need to pre-set the `startUrl` property value to `null` here. 

But besides that, Good Job! The type annotation `?string` is definitely the right one, because `startUrl` can be missing (and it will be `undefined`) and by using this annotation we are saying to flow that the `startUrl` property is "maybe a string"!